content/_releases/22/16.md

```diff
@@ -43,8 +43,8 @@ tenant configuration flag called `forced_totp` to switch this feature on.
 *   If you remove the `enabled_totp` feature flag, system will remove `forced_totp` as well.
```

Comment:
This really unclear to me - does it belong to the flow start feature above or is it a separate change?

If separate it probably needs a title etc.

Reply:
This is not a separate part it belongs to the title ### Enforce 2FA use